tests/integration/targets/problem_with_mapping/tasks/main.yml

```diff
@@ -0,0 +1,408 @@
+---
+- environment:
+    SN_HOST: "{{ sn_host }}"
+    SN_USERNAME: "{{ sn_username }}"
+    SN_PASSWORD: "{{ sn_password }}"
+
+  vars:
+    mapping:
+      problem:
+        state:
+          101: "my_new"
+          102: "asses"
+          103: "root_cause"
+          104: "progress"
+          106: "resolved"
+          107: "closed"
+
+  block:
+    - name: Retrieve problem records
```

Comment:
I think we can drop this test since gathering information about all problem records is a time-consuming operation. We might want use a query-based / filter-based task.